src/components/App/App.js

```diff
@@ -18,22 +19,30 @@ class App extends Component {
   render() {
     return (
       <main>
-      <Route
-        exact
-        path="/"
-        render={() => (
-          <MovieContainer
-            movies={this.props.movies}
-          />
-        )}
-      />
+        <Nav
+          userInfo={this.props.userInfo}
+        />
+        <Route
+          exact
+          path="/"
+          render={() => (
+            <MovieContainer
+              movies={this.props.movies}
+            />
+          )}
+        />
       </main>
     )
   }
 }
 
 const mapStateToProps = (state) => ({
-  movies: state.movies
+  movies: state.movies,
+  userInfo: {
+    id: '',
+    name: 'Ya Boi',
+    email: 'foo@yaa.com'
```

Comment:
This is good that you added a test object to render the correct userInfo data.  This setup of this object matches the Post Request object where when the correct credentials are entered in the login page, it will return the user info in an object with id, name, and email.  Once we can get the login page set up correctly, we can change the test object into the correct  user info in the API.